src/core/issues-notes.js

```diff
@@ -118,7 +118,37 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          var labelListSpan = document.createElement('span');
+          Array.from(ghIssue.labels)
+            .forEach((label, index) => {
+              var labelSpan = document.createElement('span');
```

Comment:
Labels should be `a` elements, as the value of labels is that they group issues together. Clicking on a label should take you to github, so you can see other issues with the same label.